jira-dependency-graph.py

```diff
@@ -217,7 +232,11 @@ def main():
         auth = options.cookie
     else:
         # Basic Auth is usually easier for scripts like this to deal with than Cookies.
-        auth = (options.user, options.password)
+        user = options.getuser if options.user is not None \
+                    else getpass.getpass('Username: ')
```

Comment:
we could actually use: `raw_input('Username: ')`
at the end of the day, the username is no secret, is it?